clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -425,4 +425,12 @@ DPCT_NON_ENUM_OPTION(
         "Query functionally compatible SYCL API to migrate CUDA API."),
     llvm::cl::value_desc("api"), llvm::cl::cat(DPCTCat), llvm::cl::Optional)
 
+DPCT_NON_ENUM_OPTION(
+    DPCT_OPT_TYPE(static llvm::cl::opt<bool>), UsePureSyclQueue,
+    "use-pure-sycl-queue",
+    llvm::cl::desc("Generates pure SYCL code to get queue and device. Notice "
+                   "that the runtime behavior may not be the same with the "
```

Comment:
the notice is not clear enough? could it be more specific on sycl::queue like passing it to functions?

Reply:
fixed